src/navigator/TabNavigator.tsx

```diff
@@ -3,8 +3,8 @@ import { NativeStackHeaderProps, NativeStackScreenProps } from '@react-navigatio
 import React from 'react'
 import { useTranslation } from 'react-i18next'
 import { StyleSheet } from 'react-native'
-import DAppsExplorerScreenSearchFilter from 'src/dappsExplorer/DAppsExplorerScreenSearchFilter'
-import WalletHome from 'src/home/WalletHome'
+import TabDiscover from 'src/dappsExplorer/TabDiscover'
+import TabHome from 'src/home/TabHome'
```

Comment:
thoughts on consolidating the screen components into `src/tabs/TabXXX`?

Reply:
I think it makes sense to keep them separate given all the `WalletHome.tsx` is currently in `src/home` and there are a bunch of related components in that folder. A similar clustering is occurring with `DAppsExplorerScreenSearchFilter.tsx` in `src/dappsExplorer`. While they are both tab screens it might be cluttered in the `src/tabs` folder with all their related components.

Reply:
I was just thinking of moving the screen itself there and leave the subcomponents for now in home / tokens / dapps. But this can happen later too.